action/protocol/staking/candidate_buckets_indexer.go

```diff
@@ -56,32 +80,45 @@ func (cbi *CandidatesBucketsIndexer) PutCandidates(height uint64, candidates *io
 	if err != nil {
 		return err
 	}
-	return cbi.kvStore.Put(StakingCandidatesNamespace, byteutil.Uint64ToBytesBigEndian(height), candidatesBytes)
+	if err := cbi.kvStore.Put(StakingCandidatesNamespace, byteutil.Uint64ToBytesBigEndian(height), candidatesBytes); err != nil {
```

Comment:
nit: latestHeight = byteutil.Uint64ToBytesBigEndian()
and use latestHeight below